src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GACaseManagementCategory.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.civil.model.genapplication;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.civil.model.common.Element;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class GACaseManagementCategory {
+
+    private GACaseManagementCategoryElement value;
+    @SuppressWarnings("checkstyle:MemberName")
+    private List<Element<GACaseManagementCategoryElement>> list_items;
```

Comment:
I think its better to avoid name of properties with underscore "_".

Reply:
Hey Azam, I think it's we better keep the same variable naming convention as civil.